web/src/main/ui/app/components/flows-new/edit-flow/ui/new-step-dialog-ui.component.html

```diff
@@ -45,7 +45,7 @@ <h1 mat-dialog-title>{{title}}</h1>
           </mat-panel-title>
         </mat-expansion-panel-header>
 
-        <mat-form-field *ngIf="newStepForm.value.type != 'ingestion'">
+        <mat-form-field *ngIf="newStepForm.value.stepDefinitionType !== 'INGESTION'">
```

Comment:
The whole idea of using StepType enum is to remove 'INGESTION' here (and everywhere in the code) and use StepType.INGESTION instead.

Reply:
I missed this one. Thanks!

Reply:
Oh sorry, somehow I was looking into your old commit. I see you actually substituted the string usages.)))